### ISSUE TITLE
Update Spanish translations

### ISSUE THREAD

**Issue by lazarusA** [squidfunk/mkdocs-material#4932]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### Language availability

- [X] I've checked the [list of available languages](https://bit.ly/33vFDD0)

### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "es",
  "action.edit": "Editar esta página",
  "action.skip": "Saltar a contenido",
  "action.view": "Ver código fuente de esta página",
  "announce.dismiss": "No mostrar esto de nuevo",
  "blog.archive": "Archivo",
  "blog.categories": "Categorías",
  "blog.categories.in": "en",
  "blog.continue": "Seguir leyendo",
  "blog.draft": "Borrador",
  "blog.index": "Regresar al índice",
  "blog.meta": "Metadata",
  "blog.references": "Enlaces relacionados",
  "clipboard.copy": "Copiar al portapapeles",
  "clipboard.copied": "Copiado al portapapeles",
  "consent.accept": "Aceptar",
  "consent.manage": "Gestionar cookies",
  "consent.reject": "Rechazar",
  "footer": "Pie",
  "footer.next": "Siguiente",
  "footer.previous": "Anterior",
  "header": "Cabecera",
  "meta.comments": "Comentarios",
  "meta.source": "Fuente",
  "nav": "Navegación",
  "readtime.one": "1 minuto de lectura",
  "readtime.other": "# minutos de lectura",
  "rss.created": "Fuente RSS",
  "rss.updated": "Fuente RSS de contenido actualizado",
  "search": "Buscar",
  "search.placeholder": "Búsqueda",
  "search.share": "Compartir",
  "search.reset": "Limpiar",
  "search.result.initializer": "Inicializando búsqueda",
  "search.result.placeholder": "Teclee para comenzar búsqueda",
  "search.result.none": "No se encontraron documentos",
  "search.result.one": "1 documento encontrado",
  "search.result.other": "# documentos encontrados",
  "search.result.more.one": "1 más en esta página",
  "search.result.more.other": "# más en esta página",
  "search.result.term.missing": "Falta",
  "select.language": "Seleccionar idioma",
  "select.version": "Seleccionar versión",
  "source": "Ir al repositorio",
  "source.file.contributors": "Contribuidores",
  "source.file.date.created": "Creado",
  "source.file.date.updated": "Última actualización",
  "tabs": "Pestañas",
  "toc": "Tabla de contenidos",
  "top": "Volver al principio"
}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Thanks for providing the translations! Added in 664af42367c696abf0d7f58e3903fb44579cde89.

**Comment by squidfunk** [comment]
Released as part of [9.0.8](https://github.com/squidfunk/mkdocs-material/releases/tag/9.0.8).
